README.md

```diff
@@ -12,7 +12,7 @@ the image, you need to build it with either a local docker installation or confi
 This will create the following docker images:
  * openjdk8:8-jre
  * jetty9:9.3.x
- * jetty9-compatible:1.9.x
+ * jetty9-compat
```

Comment:
should this include a label like the other two?   or perhaps remove the other labels? or perhaps make them variables like `${jetty.version}` and `${appengine.api.version}`?